src/main.ts

```diff
@@ -17,6 +18,9 @@ async function bootstrap() {
   );
   app.enableCors();
   app.register(Fastify);
+
+  app.useGlobalPipes(new ValidationPipe());
```

Comment:
```suggestion
  app.useGlobalPipes(new ValidationPipe({ whitelist: true }));
```
Je te conseille de faire ça, ça fait en sorte que tous les champs que tu ne précise pas dans ta validation sois retirés du payload d'entrée

Reply:
Ah yes je connais, en fait je le fais pas encore parce que tous les DTOs de l'appli n'ont pas encore d'annotations de class-validator et donc ça enlève tous leurs champs !